phing/tasks/deploy.xml

```diff
@@ -139,6 +139,16 @@
     </if>
   </target>
 
+  <target name="deploy:id" description="Writes deployment id information to artifact.">
+    <if>
+      <isset property="deploy.tag"/>
+      <then>
+        <!-- Write tag name for the benefit of environments like ACE that can't access it. -->
+        <exec dir="${deploy.dir}" command="echo '${deploy.tag}' > deploy_id.txt" logoutput="true" checkreturn="true" level="${blt.exec_level}"/>
```

Comment:
I suggest that we name this file `deployment_identifier` rather than `deploy_id.txt` for consistency.